srcjs/shinyapp.js

```diff
@@ -631,7 +631,8 @@ var ShinyApp = function() {
   });
 
   addMessageHandler('config', function(message) {
-    this.config = message;
+    this.config = {workerId: message.workerId, sessionId: message.sessionId};
+    if (message.user != null) exports.user = message.user;
```

Comment:
`!=` is on purpose (instead of `!==`) because it checks for both `null` and `undefined` (see [here](http://stackoverflow.com/questions/2647867/how-to-determine-if-variable-is-undefined-or-null))